### ISSUE TITLE
When using QSelect, the autocomplete dropdown won't show up if the QSelect is moved upon clicking, since it listens for the keyup event

### ISSUE THREAD

**Issue by x-ji** [quasarframework/quasar#6962]
**Describe the bug**
When using QSelect, the autocomplete dropdown won't show up if the QSelect moves upon clicking.

This seems to be caused by the fact that the dropdown only shows upon `keyup` event instead of `keydown`. See https://github.com/quasarframework/quasar/blob/96c1aa62b7564c5da844a870269ca8e9a3e990cd/ui/src/components/select/QSelect.js#L348

**Codepen/jsFiddle/Codesandbox (required)**
https://codepen.io/xji/pen/XWmZrEr

**To Reproduce**
1. Click on the top part of the `QSelect`
2. The QSelect is focused, but since it has moved, and the original place where it was clicked is no longer part of the QSelect, the `keyup` event is not triggered, and thus the autocomplete dropdown is not shown.

**Expected behavior**
The autocomplete dropdown should be triggered upon clicking on QSelect all the time, e.g. via the `keydown` event instead of the `keyup` event.

**Screenshots**
<img width="219" alt="Screenshot 2020-05-06 07 06 33" src="https://user-images.githubusercontent.com/2999209/81140035-3e21e180-8f68-11ea-862d-37022942cde0.png">

**Platform (please complete the following information):**
OS: MacOS
Node: 12.16.2
NPM: 6.14.4
Yarn: 1.22.4
Browsers: Firefox Developer Edition 76.0b8

**Additional context**

**Comment by pdanpdan** [comment]
Hello.
Clicking does not trigger neither keydown nor keyup.
I understand your problem, but it's normal there should be no interaction with the control if you move the control in another place between pointer down and pointer up.
The click is defined as a sequence of pointer down and pointer up on the same element.

**Comment by x-ji** [comment]
Sure, I understand the reasoning. Though I wonder if it would be possible to show the dropdown menu as a "drop up" in the upwards direction instead. Then this problem would also be solved and potentially some other use cases would also be covered. I opened a feature request. See also [this SO question](https://stackoverflow.com/questions/7814186/drop-down-menu-that-opens-up-upward-with-pure-css?noredirect=1&lq=1) about a dropup.

I tried to see if I can submit a PR for this but it was a bit complicated to navigate the source code for the first time (I'm also more of a backend developer than a frontend dev so sorry for confusion in some parts of my question).
